cpp/modmesh/view/RApplication.cpp

```diff
@@ -37,11 +37,13 @@
 namespace modmesh
 {
 
-RApplication * RApplication::initialize(int argc, char ** argv)
+RApplication * RApplication::initialize(int & argc, char ** argv)
```

Comment:
@tai271828 @tigercosmos This is way it crashes on Linux (Ubuntu).

@tychuang1211 It is very interesting that it does not crash with macos and Windows!